src/mbgl/style/style_parser.cpp

```diff
@@ -242,16 +241,21 @@ void StyleParser::parseSources(const JSValue& value) {
     }
 }
 
-std::unique_ptr<mapbox::geojsonvt::GeoJSONVT> StyleParser::parseGeoJSON(const JSValue& value) {
+std::unique_ptr<mapbox::geojsonvt::GeoJSONVT> StyleParser::parseGeoJSON(const JSValue& value, std::unique_ptr<SourceInfo>& info) {
     using namespace mapbox::geojsonvt;
 
+    Options options;
+    options.buffer = util::EXTENT / util::tileSize * 128;
+    options.extent = util::EXTENT;
+    info->maxZoom = options.maxZoom;
+
     try {
-        return std::make_unique<GeoJSONVT>(Convert::convert(value, 0));
+        return std::make_unique<GeoJSONVT>(Convert::convert(value, 0), options);
```

Comment:
I changed this in geojsonvt, released a new version and switched to that. Is there anything else that needs to be changed before merging?